<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-B98V00F" failures="1" tests="1" name="FourTest" time="38.257" errors="0" timestamp="2021-02-10T08:36:39 GMT-08:00">
  <testcase classname="tests.FourTest" name="testFour" time="36.031">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.FourTest.testFour(FourTest.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testFour -->
</testsuite> <!-- FourTest -->
